src/minlog.js

```diff
@@ -103,4 +103,16 @@ export default class MinLog {
       listener({entry, logger: this, rawEntry});
     });
   }
+
+  async trackTime(label, fn) {
+    let entry = {
+      _timeStart: new Date()
+    };
+    this.debug(label, entry);
```

Comment:
Why specifically `debug` and not something else?

Reply:
Want to send the logging fn as an arg?

Reply:
> Want to send the logging fn as an arg?

Naah. I would still leave a default value and then I would need to override it on every usage - not nice.

I would choose out of two options: 
1. Rename function to `debugTrackTime` and set it in stone (so-so).
2. Add a property to `MinLog` class like `trackTimeLevelName` with some reasonable default like `debug`.

Reply:
or, since it's a question of levels, I added a `time` level